### ISSUE TITLE
CsvExampleGen output to GCS

### ISSUE THREAD

**Issue by JeffersonH44** [tensorflow/tfx#552]
How can I put an output to GCS using CsvExampleGen? I have the following code:

```python
from tfx.components.base.base_component import ComponentOutputs
from tfx.proto import example_gen_pb2
from tfx.components.example_gen.csv_example_gen.component import CsvExampleGen
from tfx.utils.dsl_utils import csv_input
...

_output_bucket = 'gs://my_bucket/tfx'

# source data
_input_data = os.path.join(_output_bucket, 'data')

# output data
_output_data = os.path.join(_output_bucket, 'splitted')
train_type = TfxType('ExamplesPath', split='train')
train_type.uri = _output_data
eval_type = TfxType('ExamplesPath', split='eval')
eval_type.uri = _output_data

## channel setup
output_channel = Channel(
    type_name='ExamplesPath',
    static_artifact_collection=[train_type, eval_type]
)

...

def create_pipeline():
  examples = csv_input(_input_data)

  output = example_gen_pb2.Output(
             split_config=example_gen_pb2.SplitConfig(splits=[
                 example_gen_pb2.SplitConfig.Split(name='train', hash_buckets=5),
                 example_gen_pb2.SplitConfig.Split(name='eval', hash_buckets=1),
             ]))

  example_gen = CsvExampleGen(
      input_base=examples,
      output_config=output,
      outputs=ComponentOutputs({
          'examples': output_channel
      })
    )

```

but the bucket is empty after the `CsvExampleGen` component runs (and it ends with success). Also I'm running everything on Cloud Dataflow.

I'm using:
```bash
tensorflow==1.14.0
tfx==0.13.0
apache-airflow==1.10.4
```

**Comment by JakeTheWise** [comment]
Probably isn't the issue but looks like you have a typo where you intend to set `eval_type`'s `uri`.

**Comment by rmothukuru** [comment]
@JeffersonH44 ,
Can you please let us know if @JakeTheWise's comment helped you. Thanks!

**Comment by JeffersonH44** [comment]
@JakeTheWise yes, I tested it and is not the issue. Thanks!

**Comment by rmothukuru** [comment]
@JeffersonH44 ,
Can you please confirm if your issue is resolved. Thanks!

**Comment by JeffersonH44** [comment]
No

**Comment by rmothukuru** [comment]
Couldn't assign to @charlesccychen.

**Comment by charlesccychen** [comment]
Hey @JeffersonH44, we recently released version 0.14 RC 1 (see here: https://pypi.org/project/tfx/#history).  Can you install tfx==0.14.0rc1 and try again?  Also, does this work with the local Beam DirectRunner?

**Comment by rmothukuru** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
